artist/forms.py

```diff
@@ -22,16 +22,39 @@ class ArtistApplyForm(forms.Form):
         )
     )
     campaign_reason = forms.CharField(
-        label="Why are you raising money?",
+        label="What are you raising money for?",
         widget=forms.Textarea(
             attrs={"placeholder": "We are trying to record our album..."}
         ),
     )
-    campaign_expenses = forms.CharField(
-        label="What do you need the money for?",
-        widget=forms.Textarea(
-            attrs={"placeholder": "Mixing, mastering, studio time, etc..."}
-        ),
+    amount_raising = forms.CharField(
+        label="Amount Raising", widget=forms.TextInput(attrs={"placeholder": "$1,000"})
+    )
+    giving_back = forms.CharField(
+        label="% Back To Investors",
+        widget=forms.TextInput(attrs={"placeholder": "50%"}),
+    )
+    campaign_start = forms.CharField(
+        label="Campaign Start Date",
+        widget=forms.TextInput(attrs={"placeholder": "1/17/20"}),
+    )
+    campaign_length = forms.CharField(
+        label="Campaign Length",
+        widget=forms.TextInput(attrs={"placeholder": "30 Days, 60 days, 120 days..."}),
+    )
+    term_link = forms.CharField(
+        label="Payback Commitment",
```

Comment:
This label alone does not provide enough context to me to understand what this is. Should we provide more context?